com.unity.render-pipelines.universal/Runtime/2D/Light2D.cs

```diff
@@ -206,16 +242,7 @@ private void LateUpdate()
             if (m_LightType == LightType.Global)
                 return;
 
-            // Mesh Rebuilding
-            if (LightUtility.CheckForChange(m_ShapeLightFalloffSize, ref m_PreviousShapeLightFalloffSize) ||
```

Comment:
Found that UpdateMesh was called N times if N parameters are different as || early exits during each check. So we were generating same mesh multiple times even if parameters are same. Fixed above.

Reply:
Nice catch!